### PR TITLE
Bly 1157

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: apps/root/src/frame/components/connect-wallet-option-modal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                  <FormattedMessage
+                    description="loginOptionWalletTitle"
+                    defaultMessage="Directly connect your wallet"
+                  />
+                </Typography>
+                <WalletIcon fontSize="inherit" />
+                <Typography variant="body1">
+                  <FormattedMessage
+                    description="loginOptionPrivyDesc"
+                    defaultMessage="You wont get the amazing benefits of multi-wallet but its not privy :shrug:"
```
Comment (th0rOdinson) [inline apps/root/src/frame/components/connect-wallet-option-modal/index.tsx:73]: Shouldn't this new messages be compiled?

File: apps/root/src/services/meanApiService.ts [hunk 2]
```diff
@@ -42,14 +44,22 @@ export default class MeanApiService {
...
+  constructor(
+    contractService: ContractService,
+    axiosClient: AxiosInstance,
+    providerService: ProviderService,
+    accountService: AccountService
+  ) {
     this.axiosClient = axiosClient;
     this.contractService = contractService;
     this.providerService = providerService;
+    this.accountService = accountService;
```
Comment (th0rOdinson) [inline apps/root/src/services/meanApiService.ts:62]: With the implementation of `accountService`, what are your thoughts on removing the `accountId` parameter from the six methods associated with labels and the contact list?

**Review by th0rOdinson — APPROVED** [review]

File: apps/root/src/frame/components/connect-wallet-option-modal/index.tsx (hunk 1 above)
Comment (FiboApe) [reply apps/root/src/frame/components/connect-wallet-option-modal/index.tsx:73]: Not really, this text wont be there once we get the ones from the designer

File: apps/root/src/services/meanApiService.ts (hunk 2 above)
Comment (FiboApe) [reply apps/root/src/services/meanApiService.ts:62]: Good thinking! Will implement

**Pull request merged**